src/app/saga.test.ts

```diff
@@ -96,16 +99,16 @@ describe('App saga', () => {
   })
 
   it('Handles openScreen deep link with safe origin', async () => {
-    const deepLink = `celo://wallet/openScreen?screen=${Screens.FiatExchangeOptions}&isCashIn=true`
+    const deepLink = `celo://wallet/openScreen?screen=${Screens.FiatExchangeCurrency}&flow=CashIn`
```

Comment:
@MuckT Do we have any deeplinks going to this component? I changed the name of the component and the router props so those will break if we have any

Reply:
@jh2oman - I no active or scheduled campaigns are currently using the `FiatExchangeOptions` deep link. I've also let @laurencedelisle know so we can adjust going forward.